### PR TITLE
docs: update migrating plan

### PR DESCRIPTION
**Pull request by stevenGravy**
Updates:
- mention auto upgrading as something that can be incorporated
- mention to update repo channels and install enterprise version
- update to lower or same version for Teleport Enterprise.

### CODE REVIEW

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 1]
```diff
@@ -72,14 +72,23 @@ migrating from Teleport Enterprise to Teleport Community Edition.
...
+   # check for older versions
    $ tctl inventory ls --older-than=<version>
+   # check for newer versions
+   $ tctl inventory ls --newer-than=<version>
    ```
 
+1. Both the self-hosted and Teleport Enterprise (Cloud) support automatic upgrading
+   that is recommended to maintain a healthy set of agents. See how to 
+   [Set up Automatic Agent Updates](../upgrading/automatic-agent-updates.mdx) to
+   incorporate this into your migration.
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:90]:
```suggestion
1. We recommend that Teleport Enterprise (Self-Hosted) customers set up automatic updates to maintain a healthy set of Agents, and require this in Teleport Enterprise (Cloud). See how to 
   [Set up Automatic Agent Updates](../upgrading/automatic-agent-updates.mdx) to
   incorporate this into your migration.

```

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 2]
```diff
@@ -219,6 +228,19 @@ To migrate Teleport agents:
 
 1. Stop Teleport services on each Agent (if applicable).
 
+1. If you are using Teleport repositories to install on Linux
+   update the repository channel to `stable/cloud` for Teleport
+   Enterprise (cloud). For self-hosted you can use the major version
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:233]:
```suggestion
   Enterprise (Cloud). For self-hosted Teleport Enterprise you can use the major version
```

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 3]
```diff
@@ -219,6 +228,19 @@ To migrate Teleport agents:
 
 1. Stop Teleport services on each Agent (if applicable).
 
+1. If you are using Teleport repositories to install on Linux
+   update the repository channel to `stable/cloud` for Teleport
+   Enterprise (cloud). For self-hosted you can use the major version
+   `stable/v(=teleport.major_version=)` or if you are using the automatic
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:234]:
```suggestion
   `stable/v(=teleport.major_version=)` or if you are using automatic

```

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 4]
```diff
@@ -219,6 +228,19 @@ To migrate Teleport agents:
 
 1. Stop Teleport services on each Agent (if applicable).
 
+1. If you are using Teleport repositories to install on Linux
+   update the repository channel to `stable/cloud` for Teleport
+   Enterprise (cloud). For self-hosted you can use the major version
+   `stable/v(=teleport.major_version=)` or if you are using the automatic
+   upgrades the `stable/rolling` has all released versions.
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:235]:
```suggestion
   upgrades the `stable/rolling` channel has all released versions.

```

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 5]
```diff
@@ -219,6 +228,19 @@ To migrate Teleport agents:
 
 1. Stop Teleport services on each Agent (if applicable).
 
+1. If you are using Teleport repositories to install on Linux
+   update the repository channel to `stable/cloud` for Teleport
+   Enterprise (cloud). For self-hosted you can use the major version
+   `stable/v(=teleport.major_version=)` or if you are using the automatic
+   upgrades the `stable/rolling` has all released versions.
+
+1. If you are using the community version of Teleport uninstall `teleport`
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:237]:
```suggestion
1. If you are using the Teleport Community Edition, uninstall `teleport`

```

Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:237]: I would link to the instructions for uninstalling Teleport here

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 6]
```diff
@@ -219,6 +228,19 @@ To migrate Teleport agents:
...
+   Enterprise (cloud). For self-hosted you can use the major version
+   `stable/v(=teleport.major_version=)` or if you are using the automatic
+   upgrades the `stable/rolling` has all released versions.
+
+1. If you are using the community version of Teleport uninstall `teleport`
+   and install the enterprise version. You can confirm if the `teleport` binary
+   is enterprise by running `teleport version`. The output will contain the words
+   `Teleport Enterprise`. The enterprise version of the `teleport`
+   binary should be used when connecting to Teleport Enterprise (cloud) or
+   Teleport Enterprise self-hosted.
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:242]:
```suggestion
   binary should be used when connecting to Teleport Enterprise (Cloud) or
   Teleport Enterprise (Self-Hosted).

```

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 7]
```diff
@@ -269,6 +291,8 @@ To migrate Teleport agents:
    # delete the state secret
    kubectl -n <namespace> delete secret <release-name>-0-state
    ```
+   The settings for the new Teleport Kubernetes agent should have the
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:294]:
```suggestion
   The settings for the new Teleport Kubernetes Agent should have the 

```

**Review by ptgott — COMMENTED** [review]
Left suggestions for grammar and consistent wording in the docs

File: docs/pages/admin-guides/migrate-plans.mdx (hunk 5 above)
Comment (stevenGravy) [reply docs/pages/admin-guides/migrate-plans.mdx]: > I would link to the instructions for uninstalling Teleport here

thanks, updated

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 8]
```diff
@@ -219,6 +228,21 @@ To migrate Teleport agents:
...
+   `stable/v(=teleport.major_version=)` or if you are using automatic
+   upgrades the `stable/rolling` channel has all released versions.
+
+
+1. If you are using the Teleport Community Edition, [uninstall Teleport](./management/admin/uninstall-teleport.mdx)
+   and install the enterprise version. You can confirm if the `teleport` binary
+   is enterprise by running `teleport version`. The output will contain the words
+   `Teleport Enterprise`. The enterprise version of the `teleport`
+   binary should be used when connecting to Teleport Enterprise (Cloud) or
+   Teleport Enterprise (Self-Hosted).
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:243]:
```suggestion
1. If you are using Teleport Community Edition, [uninstall Teleport](./management/admin/uninstall-teleport.mdx)
   and install the enterprise edition of the `teleport` binary. You can confirm if the binary
   is correct by running `teleport version`. The output will contain the words
   `Teleport Enterprise`. The enterprise edition of the `teleport`
   binary should be used when connecting to Teleport Enterprise (Cloud) or
   Teleport Enterprise (Self-Hosted).

```

File: docs/pages/admin-guides/migrate-plans.mdx [hunk 9]
```diff
@@ -219,6 +228,21 @@ To migrate Teleport agents:
...
+   update the repository channel to `stable/cloud` for Teleport
+   Enterprise (Cloud). For self-hosted Teleport Enterprise you can use the major version
+   `stable/v(=teleport.major_version=)` or if you are using automatic
+   upgrades the `stable/rolling` channel has all released versions.
+
+
+1. If you are using the Teleport Community Edition, [uninstall Teleport](./management/admin/uninstall-teleport.mdx)
+   and install the enterprise version. You can confirm if the `teleport` binary
+   is enterprise by running `teleport version`. The output will contain the words
+   `Teleport Enterprise`. The enterprise version of the `teleport`
```
Comment (ptgott) [inline docs/pages/admin-guides/migrate-plans.mdx:241]: Should `teleport` here be `tctl`, since we're talking about connecting to a Teleport cluster?

**Review by ptgott — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

**Review by scottpgallagher — APPROVED** [review]
Approved.

**Pull request merged**